.github/dependabot.yml

```
@@ -9,3 +10,24 @@ updates:
     labels:
       - area/dependencies
       - area/ci
+  # keycloak UI dependencies
+  - package-ecosystem: npm
+    directory: /themes/src/main/resources/theme/keycloak/common/resources/
+    schedule:
+      interval: daily
+      time: "00:01"
+      timezone: UTC
```

This seems like too much information and it runs a little too often for my taste. Let's keep this the same as the Github Actions check.
```suggestion
      interval: weekly
```